tests/monotouch-test/Network/NWProxyConfigTests.cs

```diff
@@ -0,0 +1,185 @@
+#if !__WATCHOS__
+using System;
+using System.Collections.Generic;
+using Foundation;
+using Network;
+
+using NUnit.Framework;
+
+#nullable enable
+
+namespace MonoTouchFixtures.Network {
+
+	[TestFixture]
+	[Preserve (AllMembers = true)]
+	public class NWProxyConfigTests {
+		NWRelayHop? hop;
+		NWEndpoint? endpoint;
+		NWProxyConfig? config;
+
+		[OneTimeSetUp]
+		public void Init () => TestRuntime.AssertXcodeVersion (15, 0);
+
+		[SetUp]
+		public void SetUp ()
+		{
+			endpoint = NWEndpoint.Create ("https://github.com");
+			hop = NWRelayHop.Create (endpoint, null, null);
+			config = NWProxyConfig.CreateHttpConnect (endpoint, null);
+		}
+
+		[TearDown]
+		public void TearDown ()
+		{
+			config?.Dispose ();
+			endpoint?.Dispose ();
+			hop?.Dispose ();
+		}
+
+		[Test]
+		public void CreateRelayTest ()
+		{
+			config = null;
```

Comment:
This should use a local variable instead of overwriting the instance variable, because otherwise you may get random behavior depending on the order tests are executed (say this test fails because `config` was assigned null - then all other tests that use the instance `config` variable will also fail).